src/components/LaunchFilter/types/LaunchResponse.ts

```diff
@@ -0,0 +1,7 @@
+import type { Launch } from '../../../shared/api/types/Launch';
+
+export interface LaunchResponse {
+  data: {
+    results: Launch[];
+  };
+}
```

Comment:
why this type is in shared?

Reply:
I want to refactor the LaunchDetailsPage component  further and  use this type there

Reply:
Is that type will be used somewhere beside LaunchDetailsPage?

Reply:
Yes, here (in LaunchFilter component), it will be extends in LaunchList component and I want to use it in LaunchDetailsPage component